mongo/main.tf

```diff
@@ -0,0 +1,62 @@
+resource "mongodbatlas_project" "main" {
+  name   = var.atlas_project_name
+  org_id = var.atlas_org_id
+}
+
+resource "mongodbatlas_cluster" "cluster" {
+  project_id = mongodbatlas_project.main.id
+  name       = var.cluster_name
+
+  # Provider Settings "block"
+  provider_name = "TENANT"
+
+  # options: AWS AZURE GCP
+  backing_provider_name = "AWS"
+
+  # options: M2/M5 atlas regions per cloud provider
+  # GCP - CENTRAL_US SOUTH_AMERICA_EAST_1 WESTERN_EUROPE EASTERN_ASIA_PACIFIC NORTHEASTERN_ASIA_PACIFIC ASIA_SOUTH_1
+  # AZURE - US_EAST_2 US_WEST CANADA_CENTRAL EUROPE_NORTH
+  # AWS - US_EAST_1 US_WEST_2 EU_WEST_1 EU_CENTRAL_1 AP_SOUTH_1 AP_SOUTHEAST_1 AP_SOUTHEAST_2
+  provider_region_name = upper(var.region)
+
+  # options: M0 M2 M5
+  provider_instance_size_name = var.cluster_instance_size
+
+  # Will not change till new version of MongoDB but must be included
+  mongo_db_major_version       = "5.0"
```

Comment:
I think Mongo 6 has been released - should we move up to that?

Reply:
Ah weird - it looks like 6 isn't available for M0? It has no problem creating it - but it then wants to try and update to 6 cause it gets created on 5